subprojects/gradle/utils/src/main/kotlin/com/avito/utils/BuildFailer.kt

```diff
@@ -7,13 +7,21 @@ val Project.buildFailer: BuildFailer
 
 interface BuildFailer {
```

Comment:
If we don't mock it in Gradle testkit, it looks like useless abstraction.

Reply:
We have `FakeBuildFailer` that stores last exception and last message

Reply:
I know, and I don't see how it helps to test. In unit tests it's no use. In integration tests we don't use it.